packages/components/src/components/autocomplete/autocomplete.jsx

```diff
@@ -91,11 +94,13 @@ const Autocomplete = React.memo(props => {
     }, [filtered_items]);
 
     React.useEffect(() => {
+        if (show_list) showDropdownList();
+        if (hide_list) hideDropdownList();
```

Comment:
here we need show_list and hide_list due to showing the dropdown list immediately (show_list) after clicking the close icon, and hiding the list for when the user clicks outside of the component since we had shown the list even though the component wasn't focused on

Reply:
I had tried to do only listen to show_list and setting it to false, but it would still show another list once you select a PM, even though it had been set to false... so we need to always listen to if show_list is true or hide_list is true